beancount_web/application.py

```diff
@@ -24,6 +24,15 @@
 app.config['ASSETS_MANIFEST'] = None
 assets.init_app(app)
 
+app.api = BeancountReportAPI()
+
+app.config.raw = configparser.ConfigParser()
+user_config_defaults_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'default-settings.conf')
+app.config.raw.readfp(open(user_config_defaults_file))
```

Comment:
Good point, only copied the code and didn't really check it. Will change it to use `with`